Lib/test/test_descr.py

```diff
@@ -5003,6 +5004,26 @@ class Child(Parent):
         gc.collect()
         self.assertEqual(Parent.__subclasses__(), [])
 
+    def test_getattribute(self):
+        # add test case for gh-103272
+        class A:
+            def __init__(self) -> None:
+                self.bar = 0
+
+            def __getattribute__(self, name):
```

Comment:
Yes, I agree with @hauntsaninja, this does not seem required. Or is it?

Reply:
It isn't required. I will simplify it